<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrayPreferences</class>
 <widget class="QMainWindow" name="TrayPreferences">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>278</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>QPacmanTray settings...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qpacmantray.qrc">
    <normaloff>:/pics/qpacmantray.svg</normaloff>:/pics/qpacmantray.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="CategoryListView" name="categoryList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TrayOptionsWidget" name="trayOptions" native="true"/>
      </item>
      <item>
       <widget class="AlpmOptionsWidget" name="alpmOptions" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CategoryListView</class>
   <extends>QListView</extends>
   <header>categorylistview.h</header>
  </customwidget>
  <customwidget>
   <class>TrayOptionsWidget</class>
   <extends>QWidget</extends>
   <header>trayoptionswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AlpmOptionsWidget</class>
   <extends>QWidget</extends>
   <header>alpmoptionswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qpacmantray.qrc"/>
 </resources>
 <connections/>
</ui>
